src/components/homepage/homepage.jsx

```diff
@@ -1,7 +1,16 @@
 import React from 'react';
-
+import { Header } from '../header/header';
+import '../css/homepage.css';
+import { HomepageBody } from './homepage-body';
+import { Footer } from '../footer/footer';
 const Homepage = () => {
-	return <div>Homepage</div>;
+	return (
+		<div className={'homepage'}>
```

Comment:
why is `className ={'homepage'}` instead of  `className = 'homepage'`?

Reply:
yes, you can do `className = 'homepage'` if you are not adding any other class in there using `template literals`

Reply:
Done